CRITICAL - 2024-11-10 15:36:01 --> Undefined variable $particularName
in APPPATH\Controllers\DatabasePPEController.php on line 780.
 1 APPPATH\Controllers\DatabasePPEController.php(780): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $particularName', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\DatabasePPEController.php', 780)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->saveInventory()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 15:51:30 --> Too few arguments to function App\Controllers\DatabasePPEController::databaseListener(), 0 passed in C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\DatabasePPEController.php on line 1839.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->databaseListener()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 15:54:05 --> Unsupported operand types: string - string
in APPPATH\Views\IIRUSP.php on line 157.
 1 SYSTEMPATH\View\View.php(226): include()
 2 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('IIRUSP', [], true)
 4 APPPATH\Controllers\ServiceController.php(236): view('IIRUSP', [...])
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ServiceController->serviceablePDF()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 16:18:27 --> Unsupported operand types: string - string
in APPPATH\Views\IIRUSP.php on line 157.
 1 SYSTEMPATH\View\View.php(226): include()
 2 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('IIRUSP', [], true)
 4 APPPATH\Controllers\ServiceController.php(236): view('IIRUSP', [...])
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ServiceController->serviceablePDF()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:13:29 --> Undefined variable $image
in APPPATH\Controllers\SigninController.php on line 118.
 1 APPPATH\Controllers\SigninController.php(118): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $image', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\SigninController.php', 118)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:13:34 --> Undefined variable $image
in APPPATH\Controllers\SigninController.php on line 118.
 1 APPPATH\Controllers\SigninController.php(118): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $image', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\SigninController.php', 118)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:15:31 --> Undefined variable $image
in APPPATH\Controllers\SigninController.php on line 118.
 1 APPPATH\Controllers\SigninController.php(118): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $image', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\SigninController.php', 118)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:15:35 --> Undefined variable $image
in APPPATH\Controllers\SigninController.php on line 118.
 1 APPPATH\Controllers\SigninController.php(118): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $image', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\SigninController.php', 118)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:15:42 --> Undefined variable $image
in APPPATH\Controllers\SigninController.php on line 118.
 1 APPPATH\Controllers\SigninController.php(118): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $image', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\SigninController.php', 118)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:16:03 --> Undefined variable $image
in APPPATH\Controllers\SigninController.php on line 118.
 1 APPPATH\Controllers\SigninController.php(118): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $image', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\SigninController.php', 118)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:19:29 --> Too few arguments to function App\Controllers\NotificationController::addNotification(), 1 passed in C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php on line 140 and exactly 2 expected
in APPPATH\Controllers\NotificationController.php on line 12.
 1 APPPATH\Controllers\SigninController.php(140): App\Controllers\NotificationController->addNotification('New user signed up: Genriel V. Del Mundo')
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:19:36 --> Too few arguments to function App\Controllers\NotificationController::addNotification(), 1 passed in C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php on line 140 and exactly 2 expected
in APPPATH\Controllers\NotificationController.php on line 12.
 1 APPPATH\Controllers\SigninController.php(140): App\Controllers\NotificationController->addNotification('New user signed up: Genriel V. Del Mundo')
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:22:46 --> Too few arguments to function App\Controllers\NotificationController::addNotification(), 1 passed in C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php on line 140 and exactly 2 expected
in APPPATH\Controllers\NotificationController.php on line 12.
 1 APPPATH\Controllers\SigninController.php(140): App\Controllers\NotificationController->addNotification('New user signed up: Ronel O. Cacho')
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:24:45 --> Too few arguments to function App\Controllers\NotificationController::addNotification(), 1 passed in C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php on line 140 and exactly 2 expected
in APPPATH\Controllers\NotificationController.php on line 12.
 1 APPPATH\Controllers\SigninController.php(140): App\Controllers\NotificationController->addNotification('New user signed up: Lorna A. Santos')
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->signup()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php(159): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SigninController->getVerify()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php(159): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SigninController->getVerify()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublN...')
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublN...')
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT DISTINCT...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\OrderController.php(178): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\OrderController->getArticles()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrderController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT DISTINCT...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\OrderController.php(178): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\OrderController->getArticles()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrderController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublN...')
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublN...')
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1000): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getInventory()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1000): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getInventory()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-11-10 18:46:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\OrderController.php(172): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\OrderController->getShop()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrderController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\OrderController.php(172): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\OrderController->getShop()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrderController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `fullname` = :fullname:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `shop`
ORDER BY `shop_rate` DESC', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\OrderController.php(172): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\OrderController->getShop()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrderController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `inventoryppe`
ORDER BY `id` DESC', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\DatabasePPEController.php(1000): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getInventory()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `notifications`
ORDER BY `id` DESC', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `token` = :token:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublNsJf5w2dMCQkKp1y6BSZo9rOvYgaIL4zDR7q')
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `token` = :token:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublNsJf5w2dMCQkKp1y6BSZo9rOvYgaIL4zDR7q')
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\SigninController.php(159): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->getVerify()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT DISTINCT `article`
FROM `shop`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\OrderController.php(178): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\OrderController->getArticles()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrderController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `fullname` = :fullname:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-10 18:46:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `notifications`
ORDER BY `id` DESC', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php(284): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SigninController->verifyPassword()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php(284): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SigninController->verifyPassword()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:44 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `token` = :token:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\SigninController.php(284): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->verifyPassword()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:48 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php(284): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SigninController->verifyPassword()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\SigninController.php(284): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SigninController->verifyPassword()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:48 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `token` = :token:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\SigninController.php(284): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SigninController->verifyPassword()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublN...')
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublN...')
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `token` = :token:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\RequestController.php(21): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\RequestController->users('x0FPVnXT3EiublNsJf5w2dMCQkKp1y6BSZo9rOvYgaIL4zDR7q')
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RequestController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `signin`
WHERE `fullname` = :fullname:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\ServiceController.php(61): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ServiceController->getDataUser()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServiceController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1901): CodeIgniter\Model->countAllResults()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getEmployeeCount()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1901): CodeIgniter\Model->countAllResults()
#5 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getEmployeeCount()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `signin`
WHERE `position` != :position:', [...], false)
 3 SYSTEMPATH\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\DatabasePPEController.php(1901): CodeIgniter\Model->countAllResults()
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getEmployeeCount()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1936): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getInventoryCount()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1936): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getInventoryCount()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `inventoryppe`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\DatabasePPEController.php(1936): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getInventoryCount()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `notifications`
ORDER BY `id` DESC', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `employe...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\SupplyController.php(136): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SupplyController->getEmployeeSupplies()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SupplyController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `employe...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\SupplyController.php(136): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\SupplyController->getEmployeeSupplies()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SupplyController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `employeesupplies`.`id`, `employeesupplies`.`item_code`, `employeesupplies`.`fullname`, `employeesupplies`.`issue_quantity`, `employeesupplies`.`no_days`, `employeesupplies`.`status`, `employeesupplies`.`issued_date`, `employeesupplies`.`end_date`, `databasesupplies`.`image`, `databasesupplies`.`item`, `databasesupplies`.`description`
FROM `employeesupplies`
JOIN `databasesupplies` ON `databasesupplies`.`item_code` = `employeesupplies`.`item_code`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\SupplyController.php(136): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\SupplyController->getEmployeeSupplies()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SupplyController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1915): CodeIgniter\Model->countAllResults()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getTotalInventory()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1915): CodeIgniter\Model->countAllResults()
#5 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getTotalInventory()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `inventoryppe`', [], false)
 3 SYSTEMPATH\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\DatabasePPEController.php(1915): CodeIgniter\Model->countAllResults()
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getTotalInventory()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `notifications`
ORDER BY `id` DESC', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\NotificationController.php(28): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\NotificationController->getNotification()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NotificationController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-10 18:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u311273531_dilg'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u311273531_dilg', 'Dilg_2024', 'u311273531_dilg', 3306, '', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1927): CodeIgniter\Model->countAllResults()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getPPECount()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\laragon\www\dilg\backend\backend\system\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1927): CodeIgniter\Model->countAllResults()
#5 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getPPECount()
#6 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-11-10 18:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1699): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `databaseppe`', [], false)
 3 SYSTEMPATH\Model.php(599): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\DatabasePPEController.php(1927): CodeIgniter\Model->countAllResults()
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->getPPECount()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
